pkg/provider/wif_configs.go

```diff
@@ -0,0 +1,56 @@
+package provider
+
+import (
+	"fmt"
+
+	"github.com/openshift-online/ocm-cli/pkg/arguments"
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+)
+
+func getWifConfigs(client *cmv1.Client) (wifConfigs []*cmv1.WifConfig, err error) {
+	collection := client.GCP().WifConfigs()
+	page := 1
+	size := 100
+	for {
+		var response *cmv1.WifConfigsListResponse
+		response, err = collection.List().
+			Page(page).
+			Size(size).
+			Send()
+		if err != nil {
+			return
+		}
+		wifConfigs = append(wifConfigs, response.Items().Slice()...)
+		if response.Size() < size {
+			break
+		}
+		page++
+	}
+
+	if len(wifConfigs) == 0 {
+		return nil, fmt.Errorf("no wif configs found")
+	}
+	return
+}
+
+func GetWifConfigs(client *cmv1.Client) (wifConfigs []*cmv1.WifConfig, err error) {
+	return getWifConfigs(client)
```

Comment:
Out of curiosity: Why did you make place all of the functionality within a private helper?